apps/examples/webserver_test/webserver_test_main.c

```diff
@@ -565,16 +568,25 @@ static int ws_test_base(void *arg)
 			}
 		}
 		int loopcount = 0;
+		int buf_msg_len = 0;
+
+		len = 0;
+		memset(param->response->message, 0, WS_TEST_CONF_MAX_MESSAGE_SIZE);
```

Comment:
how about checking param->response->message is NULL or not and size of message???
If message is allocated with smaller size than WS_TEST_CONF_MAX_MESSAGE_SIZE, it can make overflow.

Reply:
1. Added the validation for param->response->message 
2. param->response->message is initialized by ws_test_http_client_response_init() and size is always WS_TEST_CONF_MAX_MESSAGE_SIZE